source/guides/this-guide/updates-since-videos.md

```diff
@@ -0,0 +1,150 @@
+Updates Since the Tutorial Videos
+--------------
+
+Denizen has had quite a number of updates over the years since the original tutorial videos were published! This guide page documents the changes that you should know about for best practices.
+
+### Colon Syntax
+
+Braced syntax has been replaced with colon syntax. As documented on the page for [The If Command](/guides/basics/if-command), colon syntax allows you to use multiple commands within an `if` command, `foreach` command, or `while` command. Unlike braced syntax, no closing character is needed - when you return to the prior level of spacing, the script will continue as normal.
+
+
+If you've written scripts using braced syntax, you can quickly convert them to colon syntax by deleting all closing brace characters (`}`), replacing the opening brace character (`{`) with a colon, and removing any extra spaces between the colon and the last other character in the line.
+
+For example, here's a script converted from braced syntax to colon syntax:
+
+```dscript_green
```

Comment:
Read the `sample-scripts` guide and apply the color coding

Reply:
These scripts will technically work, so I made them green. I can make them red if you want, although `sample-scripts` should then be updated to indicate that they may also represent outdated syntax.

Reply:
`Scripts that are bad and should not be used have a red outline. These scripts usually demonstrate common mistakes or pitfalls, and are presented only as examples of what **NOT** to do.`
Does this not sufficiently describe the scripts in this document? They appear to be examples of what not to do.

Reply:
It is my position that the documentation, especially documentation *for* the documentation, should over-explain where it is reasonable to do so. Here, a user is going to strictly apply these rules as-written and assume that a red script is *fundamentally bad*, filling in the gaps about *why* on their own.

I think it would be an easy thing to add and would expand their understanding. A commit that resolves these reviews is forthcoming.